chainer/functions/array/get_item.py

```diff
@@ -37,7 +50,9 @@ def get_item(x, slices):
 
     Args:
         x (tuple of Variables): Variable to be sliced.
-        slices (slice or tuple of slices): Slice objects to slice variable.
+        slices (int, slice, None or Ellipsis or tuple of them): Basic slicing
```

Comment:
I wonder if the notions here such as "basic slicing" and "newaxis" are common to users. How about adding a hyperlink to http://docs.scipy.org/doc/numpy/reference/arrays.indexing.html in the docstrings?